ceph.spec.in

```diff
@@ -584,14 +584,13 @@ Requires:	librbd1 = %{_epoch_prefix}%{version}-%{release}
 Daemon for mirroring RBD images between Ceph clusters, streaming
 changes asynchronously.
 
-%package -n ceph-immutable-object-cache
```

Comment:
should be `%package immutable-object-cache` ?

Reply:
If the package name needs to be ceph-immutable-object-cache, then it should be `%package immutable-object-cache` (i.e. remove `-n` and the `ceph-` prefix, and rpm will automatically add the `ceph-` prefix).

Reply:
@runsisi agreed. after reading @tserong 's comment and rereading the rpm doc referenced in the commit message, i realized i misread the document. so instead, i am removing "-n" and "ceph-" prefix. and split the commit into two:

1. remove `-n` and the `ceph-` prefix, as put by Tim
1. remove libradospp1 runtime dependency.